build/scenarios-proxy.sh

```diff
@@ -7,6 +7,7 @@ SESSION=`date '+%Y%m%d%H%M%S'`
 proxyJobs="--config https://raw.githubusercontent.com/aspnet/Benchmarks/master/src/BenchmarksDriver2/benchmarks.proxy.yml "
 
 jobs=(
+  "--scenario proxy-yarp        $proxyJobs --property proxy=yarp"
```

Comment:
We are not using these files anymore, it's proxy-scenarios.yml

Reply:
Should we delete these?